lua/matproxy/matproxy_tardis.lua

```diff
@@ -1,12 +1,26 @@
 matproxy.Add({
-    name = "TARDIS_State",
+    name = "TARDIS_State_Texture",
```

Comment:
Do any TARDISes use this already outside of base?

Reply:
Not that I know of